packages/preview-middleware-client/src/messagebundle.properties

```diff
@@ -37,3 +37,5 @@ ADP_REUSE_COMPONENTS_MESSAGE = Reuse components are detected for some views in t
 CPE_CHANGES_VISIBLE_AFTER_SAVE_AND_RELOAD_MESSAGE = Note: The change will be visible after save and reload. 
 
 TABLE_ROWS_NEEDED_TO_CREATE_CUSTOM_COLUMN=At least one table row is required to create new custom column. Make sure the table data is loaded and try again.
+
+HIGHER_LAYER_CHANGES_INFO_MESSAGE=The application was reloaded because of changes in a higher layer.
```

Comment:
Question: Can this text really be understood by a user? I have to admit I don't know what "higher layer" actually means.

Reply:
Fixed in [a584e48](https://github.com/SAP/open-ux-tools/pull/2536/commits/a584e48923d502d5f4c5e021fd806ef74adfac63)